applications/CoSimulationApplication/python_scripts/coupling_operations/compute_boundary_force.py

```diff
@@ -0,0 +1,163 @@
+        self.interval = self.settings["interval"].GetVector()
+
+        if(self.model_part.GetCommunicator().MyPID() == 0):
+            if(self.write_output_file):
+                output_file_name = self.model_part_name + "_global_force.dat"
+                file_handler_settings = KM.Parameters(self.settings["output_file_settings"])
+                if file_handler_settings.Has("file_name"):
+                    warn_msg = 'Unexpected user-specified entry found in "output_file_settings": {"file_name": '
+                    warn_msg += '"' + file_handler_settings["file_name"].GetString() + '"}\n'
+                    warn_msg += 'Using this specififed file name instead of the default "' + output_file_name + '"'
+                    cs_tools.cs_print_info(self._ClassName(), warn_msg)
+                else:
+                    file_handler_settings.AddEmptyValue("file_name")
+                    file_handler_settings["file_name"].SetString(output_file_name)
+                file_header = self._GetFileHeader()
+                self.output_file = TimeBasedAsciiFileWriterUtility(self.model_part, file_handler_settings, file_header).file
+
+    def Execute(self):        
+        current_time = self.model_part.ProcessInfo[KM.TIME]
+
+        if((current_time >= self.interval[0]) and (current_time < self.interval[1])):
+            results = self._EvaluateGlobalForces()
+
+            if(self.model_part.GetCommunicator().MyPID() == 0):
+                output = []
+                output.extend(results)
+                output_values = [format(val, self.format) for val in output]
+                # not formatting time in order to not lead to problems with time recognition
+                # in the file writer when restarting
+                output_values.insert(0, str(current_time))
+
+                if(self.echo_level > 2):
+                    # print to screen the results at echo level 3 or higher
+                    res_labels = ['time: ', 'vel_x: ', 'vel_y: ', 'vel_z: ', 'f_x: ', 'f_y: ', 'f_z: ', 'p: ',]
+                    result_msg = 'Boundary Force force evaluation for model part ' + self.model_part_name + '\n'
+                    result_msg += ', '.join([a + b for a, b in zip(res_labels, output_values)])
+                    cs_tools.cs_print_info(self._ClassName(), result_msg)
+                
+                if(self.write_output_file):
+                    self.output_file.write(' '.join(output_values) + '\n')
+    
+    def _EvaluateGlobalForces(self):
+        # vel_x, vel_y, vel_z
+        velocity = [0.0, 0.0, 0.0]
+        sum_forces = [0.0, 0.0, 0.0]
+        pressure_list = [0.0]
+
+        # aktuellen Zeitstep zurück setzen - warum ist das nötig ?
+        # TODO !!
+        for node in self.model_part.Nodes:
+            node.SetSolutionStepValue(KM.REACTION_X, 0, 0)
+            node.SetSolutionStepValue(KM.REACTION_Y, 0, 0)
+            node.SetSolutionStepValue(KM.REACTION_Z, 0, 0)
```

Comment:
you should use the `VariableUtils` for this. I think the fct is `SetVariable`

Reply:
I addressed this, but later in:

https://github.com/KratosMultiphysics/Kratos/blob/f6204c015c6a79d05006d3d5a8d814161740d023/applications/CoSimulationApplication/python_scripts/coupling_operations/compute_boundary_force.py#L104-L106

I left them unchanged because I compute some values elementwise, and then apply them to the nodes.